Simplified Chinese README

clarkzjw opened the pull request:
I translated the README file to simplified Chinese :smile: 
Just a little bit of contribution.

clarkzjw committed:
simplified Chinese README

clarkzjw committed:
add jinweizhao :smile:

callumacrae commented:
I mean I don't understand Chinese, but this looks good to me!

alfredxing commented:
Thanks! :+1:
From what I can understand, this is a direct translation from [the 正體 version](https://github.com/muan/hello-world/blob/gh-pages/README-zhtw.md)?

clarkzjw commented:
@alfredxing Well, actually, the differences between traditional Chinese and simplified Chinese are quite small, so sometimes a direct translation is just ok.

But, for some historical reasons, there do exists differences in some words or grammar preferences between traditional Chinese and simplified Chinese.To give an example, "Computer programs" are translated into "程序" in simplified Chinese, but "程式" in traditional Chinese.

alfredxing commented:
@JinweiClarkChao Yep. But the content is basically the same, right?

muan commented on README.md:
```diff
@@ -8,6 +8,8 @@ If you're looking for resources to learn about web programming, checkout @jlord'
 
 [`正體中文的介紹檔案`](https://github.com/muan/hello-world/blob/gh-pages/README-zhtw.md)
 
+[`简体中文的介绍文件`](README-zhcn.md)
```
let's make the translations a list :), i.e. 
- first lang
- second lang

clarkzjw commented:
@alfredxing yes

clarkzjw commented:
@alfredxing Mostly, the characters used in traditional Chinese and simplified Chinese are exactly the same .

muan commented:
Thanks! I'll merge this now.
